moveit_core/robot_trajectory/src/robot_trajectory.cpp

```diff
@@ -168,10 +168,13 @@ RobotTrajectory& RobotTrajectory::unwind()
     // unwrap continuous joints
```

Comment:
@mechwiz Here's a question. No need to address in this PR but I'm just interested in what you think.

Does it make sense to only unwrap continuous joints? I think you could still have a joint jump if a revolute joint had a large limit range.

Reply:
I think I only saw this joint jump issue with continuous joints since the unwrapping was being done incorrectly. With revolute joints with large ranges, I don't think there will be any joint jump issues since they are never unwrapped. For example if the range is +/- 6PI, the current state is 6PI, and you specify a goal state of -6PI, the joint will actually move 6 full revolutions whereas for a continuous joint it will just stay as is since they are the same state. In other words, the values used for the revolute joint are absolute (and so no joint jumps will occur) whereas the continuous joint are relative (and so need to be unwinded correctly).

All this said - I could understand the usecase of a revolute joint with large limits "acting" like a continuous joint within the limits it has so that there wouldn't potentially be extra revolutions when going to a named state (which has a hardcoded value) but I guess a line had to be drawn somewhere between the two kinds of joints.